src/lib/utils.ts

```diff
@@ -315,7 +314,7 @@ export function multiplyCoin({ amount, denom }: Coin, mult: number): Coin {
 }
 
 // return true if a > b, or a undefined
-function heightGreater(a: Height | undefined, b: Height): boolean {
+export function heightGreater(a: Height | undefined, b: Height): boolean {
```

Comment:
```suggestion
export function heightGreater(a?: Height, b: Height): boolean {
```

Reply:
Nope. You cannot have a required argument after an optional argument - tsc already caught me on this one (I tried it)